content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
-destination location. To make that process easier, Odoo uses *Putaway rules*. But what is a putaway
-rule? Putaway is the process of taking products off the receiving shipments and directly putting
-them into the most appropriate location.
+*Putaway* is the process of routing products to appropriate storage locations upon shipment arrival.
+Odoo can accomplish this seamlessly using *putaway rules*, which dictate how products move through
+specified warehouse locations.
 
-If, for example, a warehouse contains volatile substances, it is important to make sure that certain
-products are not stored close to each other because of a potential chemical reaction. That’s where
-putaway rules intervene, to avoid storing products wrongly.
+Upon shipment arrival, operations are generated based on putaway rules to efficiently move products
+to specified locations and ensure easy retrieval. Putaway rules can also prevent volatile substances
+from being stored in close proximity by directing them to different locations determined by the
+warehouse manager.
 
-Configuration
-=============
+.. seealso::
+   - :ref:`Routes -- how products move<inventory/routes/use_routes/push>`
+   - :ref:`Warehouse locations<inventory/warehouse_location/definition>`
 
-In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the
-*Multi-Step Routes*. By doing so, the *Storage Locations* will be automatically activated.
+Enable putaway rules
+====================
 
-.. image:: putaway/putaw1.png
+Begin by navigating to :menuselection:`Inventory --> Configuration --> Settings`, then activate the
+:guilabel:`Multi-Step Routes` checkbox under the :guilabel:`Warehouse` heading. By doing so, the
+:guilabel:`Storage Locations` setting is automatically enabled as well. Finally, click
+:guilabel:`Save`.
+
+.. image:: putaway/multi-step-routes.png
    :align: center
+   :alt: Check multi-step routes box
+
+Put away products
+=================
+
+Putaway rules are commonly used to move products to specific locations for easy retrieval. Thus,
+products are grouped in the same :guilabel:`Product Category` when a single putaway rule captures
+the group's storage needs.
 
-Setting up a Putaway Rule
-=========================
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Use the :guilabel:`Create` button to configure a new putaway rule
+on a :guilabel:`Product` and/or :guilabel:`Product Category` that the rule affects.
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+In the same line, the :guilabel:`When product arrives in` location is where the putaway rule is
+triggered to create an operation to move the product to the :guilabel:`Store to` location. The
+:guilabel:`Store to` location must be sublocation of the first (ie. `WH/Stock/Fruits` is a specific,
```

Comment:
I would make this an "e.g." instead of an "i.e." since it's an example.
```suggestion
:guilabel:`Store to` location must be sublocation of the first (e.g. `WH/Stock/Fruits` is a specific,
```